src/styles/components/_code-block.scss

```diff
@@ -29,7 +29,7 @@ $breakpoint-tablet: 820px;
       line-height: 2.5rem; // 40px/16
 
       &::after {
-        left: 13rem;
+        left: 27rem;
```

Comment:
With the `transform` property updated earlier, we can delete the `left` property here.